etc/config/slang.defaults.properties

```diff
@@ -0,0 +1,14 @@
+compilers=&defaultslangc
+
+defaultCompiler=slangc
+supportsBinary=false
+compilerType=slang
+instructionSet=spirv
+
+disassemblerPath=/opt/compiler-explorer/SPIRV-Tools-master/build/tools/spirv-dis
+
+group.defaultslangc.compilers=slangc
+compiler.slangc.exe=/usr/bin/slangc
+compiler.slangc.name=slangc
+compiler.slangc.versionFlag=-version
```

Comment:
this can be made part of the group to apply to all slangc-compilers in future (like the suggestion in the amazon properties)